src/components/explorer/results/QueryTotalAttentionResultsContainer.js

```diff
@@ -28,6 +29,7 @@ const localMessages = {
   },
   downloadOneCsv: { id: 'explorer.attention.total.downloadCsv', defaultMessage: 'Download all story URLs' },
   downloadCsv: { id: 'explorer.attention.total.downloadCsv', defaultMessage: 'Download all story URLs for {name}' },
+  downloadSvg: { id: 'explorer.attention.total.downloadSvg', defaultMessage: 'Download SVG' },
```

Comment:
Do I need to do anything to to extract this string for i18n?

Reply:
I see! Thanks for the suggestion! Just so that I know, if I were to add a new string, do I need to extract them for translations or does this just automatically happen somehow?

Reply:
Out of curiosity, is there a plan to localize? Why go through the effort in the first place (not a judgement--just trying to understand)?